environment.yaml

```diff
@@ -0,0 +1,22 @@
+name: pyuvdata
+channels:
+  - conda-forge
```

Comment:
Should conda-forge be the first channel here? Does that usurp priority? I would think it should be used as a second priority when needed. But maybe I'm wrong.

Reply:
I copied it from the thing we run for CI. I think you're right, but we swapped it on purpose when we were having IERS issues and the default channel didn't give us the correct astropy version.

Reply:
Ah OK. Cool.